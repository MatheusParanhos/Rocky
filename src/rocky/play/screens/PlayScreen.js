import React, { Component } from "react";
import { Text, View, StyleSheet } from "react-native";
import PlayContainer from "../containers/PlayContainer";

export default class PlayScreen extends Component {
  render() {
    return (
      <View style={styles.container}>
        <PlayContainer />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "white"
  }
});
