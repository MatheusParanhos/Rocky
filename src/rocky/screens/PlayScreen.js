import React, { Component } from 'react'
import { Text, View, StyleSheet } from 'react-native'

export default class PlayScreen extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Text> Play </Text>
      </View>
    )
  }
}

const styles = StyleSheet.create({
    container: {
      flex: 1,
      justifyContent: "center",
      alignItems: "center",
      backgroundColor: 'white'
    }
  });
