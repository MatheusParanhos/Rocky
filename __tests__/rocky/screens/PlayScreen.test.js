import React from 'react';
import PlayScreen from '../../../src/rocky/screens/PlayScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<PlayScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});