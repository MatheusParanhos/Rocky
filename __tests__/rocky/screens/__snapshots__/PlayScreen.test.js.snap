// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "height": 100,
          "justifyContent": "center",
          "width": 100,
        }
      }
    >
      <Text>
        
      </Text>
      <Text>
        
      </Text>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "tomato",
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "width": 576.9230769230769,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
           
          Rock
           
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "tomato",
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "width": 576.9230769230769,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
           
          Paper
           
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "tomato",
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "width": 576.9230769230769,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
            }
          }
        >
           
          Scissors
           
        </Text>
      </View>
    </View>
  </View>
</View>
`;
